velox/docs/configs.rst

```diff
@@ -206,6 +211,12 @@ Spilling
        If the limit is zero, then the spiller always spills a previously spilled partition if it has any data. This is
        to avoid spill from a partition with a small amount of data which might result in generating too many small
        spilled files.
+   * - spill_compression_codec
+     - string
+     - none
+     - Specifies the compression algorithm type to compress the spilled data before write to disk to trade cpu for
+       spilling io efficiency. The supported compression kind includes 'zlib', 'snappy', 'lzo', 'zstd', 'lz4' and
```

Comment:
"The supported compression kind includes" -> "The supported compression codecs are: ZLIB, SNAPPY,..."